manuscript.Rmd

```diff
@@ -68,55 +68,154 @@ set.seed(42)
-
-
+Yet, despite the heavy reliance on computational infrastructure, the current academic infrastructure does not adequately incentivise software development and, specifically, good software engineering practice that we would expect from a research software engineer [@baxter2012research].
+Here, we specifically comment on the proposed evaluation scheme for research software contributions (Table 3 of @gartner2022responsible).
+While the authors criticize the use of invalid metrics to evaluate publications (Journal Impact Factors, Citations, H-Index), those same metrics (or variants thereof) are mainly relied on in their proposal to evaluate research software.
+However, they same critisism applies to those metrics in the context of research software evaluation.
+We therefore argue to follow their evaluation scheme for publications: In a first stage, rigor should be evaluated, establishing whether the software product passes a certain treshold, and in the second stage, more complicated dimensions, such as impact, can be evaluated.
+In the following, we discuss their proposed criteria.
+We then discuss rigor in the context of research software, and propose specific criteria to measure it.
+At last, we elaborate on the assessment of impact in the context of research software.
+
+# Proposed Criteria
+
+## ID 9: Citations
+@gartner2022responsible suggest the number of citations (their indicator #9 in Table 3) to evaluate the impact of a software package.
+However, citations suffer from serious shortcoming when used to evaluate the impact of research software.
+For example, it is much more commonly accepted to cite data analysis packages than supporting packages (such as papaja [@R-papaja], RMarkdown, RStudio, and LaTeX used to render this article). Therefore, the total citation count is confounded with the type of software.
+A further complication arises through the fact that functionality of successful, modular scientific software is imported, re-used, and re-exported in other software packages.
+While we highly encourage this from both a software engineering perspective and for scientific progress, it poses further difficulties to citation-based metrics.
+This especially punishes highly relied-on backends such as optimization algorithms: For example, the NLopt optimization suite is cited 1,711 times on google scholar.
+However, the algorithms from that suite are used in countless software packages, for example, as a backend of OpenMx which again has 811 citations.
+Moreover, the algorithms from that suite are all open source, and therefore serve as a reference for others.
+As an example, consider the lme4 package, that adapted one of the algortihms from NLopt.
+This package has 57,820 citations on google scholar, yet no single lme4 user probably ever cited the NLopt library.
+We therefore argue that merrit statements are much more useful to assess impact of research software.
+
+## ID 11: GitHub Stars
+We strongly object to the metric Github stars.
```

Comment:
```suggestion
We object to the use of Github stars as a general metric.
```

I think an author may use GH stars in their merit/impact statement. However, a numeric comparison is not warranted.